src/index.ts

```diff
@@ -60,6 +79,72 @@ interface WsJobStatusResponse extends JobStatusResponse {
     final?: boolean;
 }
 
+/**
+ * Represents control metadata that may be of interest to a client.
+ */
+interface Control {
+    /**
+     * The type of control this metadata describes.
+     */
+    controlType: string;
+
+    /**
+     * The context of information about a control (eg MainMap.Height). Used by a
+     * client application to infer the use/purpose of the parameter.
+     */
+    purpose: string;
+
+    /**
+     * The height of the control in millimeters.
+     */
+    height: number;
+
+    /**
+     * The width of the control in millimeters.
+     */
+    width: number;
+}
+
+interface ControlMetadata {
```

Comment:
Sorry, can we make this one `ControlResponse` and make `Control` -> `ControlMetadata`?

Reply:
Done